.github/workflows/nightly-tests.yml

```diff
@@ -56,6 +83,28 @@ jobs:
           test: suites/FullIntegration.json
           vault-token: ${{ steps.vault-token-step.outputs.vault-token }}
 
+      - name: truncate tables in the wsm db
```

Comment:
So we're cleaning before and after? I guess add (post-test) to the name to avoid confusion.

It might be helpful to have a flag to skip the cleanup in case you need to debug DB state.